### PR TITLE
Reed

### PR DESCRIPTION
**Pull request by martynrevans**
Made some changes to the Reed Learning project story including images and text and created a new layout putting the image on the left and the text on the right.

### CODE REVIEW

File: source/project-stories/reed-learning.erb [hunk 1]
```diff
@@ -164,11 +174,20 @@ related:
       <p class="highlight__large-text"></p>
       <p class="highlight__small-text"><%= current_page.data.what_we_did_4.text %></p>
     </div>
-  </section>
+  </section> -->
```
Comment (FriedSock) [inline source/project-stories/reed-learning.erb]: Don't comment things out, just delete them. If we need to refer to the code later we can just look back in the git history. This is especially important for html where people can look at this by inspecting the page source.

File: source/project-stories/reed-learning.erb [hunk 2]
```diff
@@ -197,9 +216,9 @@ related:
       <p class="highlight__large-text"></p>
       <p class="highlight__small-text"><%= current_page.data.what_we_did_8.text %></p>
     </div>
-  </section>
+  </section> -->
```
Comment (FriedSock) [inline source/project-stories/reed-learning.erb]: same here.

File: source/project-stories/reed-learning.erb (hunk 1 above)
Comment (h-lame) [inline source/project-stories/reed-learning.erb]: If we're deleteing things, it also makes sense to remove all the `what_we_did_xxx` sections from the frontmatter as they're only referenced in these commented out sections and if they're going...

**Comment by h-lame** [comment]
I hope there's an explanation of that honeycomb graphic because it is ✨ baffling ✨

**Comment by FriedSock** [comment]
Lets see what its like when its deployed to staging

**Comment by martynrevans** [comment]
It probably makes no sense (the honeycomb) but let's see what people think

**Comment by GraemeMcCubbin** [comment]
Could the honeycomb be put into an iPad graphic?

**Pull request merged**